File: Data Structures & Algorithms/missing-number/submission-1.java
class Solution {
    public int missingNumber(int[] nums) {
        int n=nums.length;
        int tsum=n*(n+1)/2;
        int acsum=0;
        for(int num:nums){
            acsum+=num;
        }
        return tsum-acsum;


        
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq_map=Counter(nums)
        buckets = [[] for _ in range(len(nums) + 1)]
        for num, freq in freq_map.items():
            buckets[freq].append(num)
        result = []
        for i in range(len(buckets) - 1, 0, -1):  # from high freq to low
            for num in buckets[i]:
                result.append(num)
                if len(result) == k:
                    return result

        
        